lib/util/include/hse_util/alloc.h

```diff
@@ -15,22 +15,7 @@
 #include <mock/allocation.h>
 #endif /* HSE_MOCKING */
 
-/**
- * alloc_aligned() - allocated aligned memory
- * @size:   desired number of bytes
- * @align:  desired alignment
- *
- * %align must be a power-of-two
- */
-/* MTF_MOCK */
-void *
-alloc_aligned(size_t size, size_t align);
-
-/* MTF_MOCK */
-void
-free_aligned(const void *ptr);
-
-#define alloc_page_aligned(_sz) alloc_aligned((_sz), PAGE_SIZE)
+#define page_aligned_alloc(_sz) aligned_alloc(PAGE_SIZE, (_sz))
```

Comment:
This should probably ensure sz is page aligned...

Reply:
Will be caught by asan in the event it isn't if that is cool with you

Reply:
Thoughts on getting rid of `page_aligned_alloc` and have callers use `aligned_alloc(PAGE_SIZE, sz)`?

Reply:
I would like that. I see no real value here.